addons/cppcheckdata.py

```diff
@@ -642,7 +642,7 @@ def astIsFloat(token):
         return False
     if token.str == '.':
         return astIsFloat(token.astOperand2)
-    if '+-*/%'.find(token.str) == 0:
+    if token.str not in '+-*/%':
```

Comment:
I believe it looks if token is an arithmetic op. so it should be `if token.str in '+-*/%'` without not. 

hmm we can almost use token.isArithmeticalOp except that isArithmeticalOp is true for << and >>.